site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -405,6 +405,39 @@ Or view the following error message in the NGINX logs:
 The request body exceeds the [client_max_body_size](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size).
 To **resolve** this, you can configure the `client_max_body_size` using the `ClientSettingsPolicy` API. Read the [Client Settings Policy]({{< relref "how-to/traffic-management/client-settings.md" >}}) documentation for more information.
 
+##### IP Family Mismatch Errors
+
+If you `describe` your HTTPRoute and see the following error:
+
+```text
+    Conditions:
+      Last Transition Time:  2024-07-14T23:36:37Z
+      Message:               The route is accepted
+      Observed Generation:   1
+      Reason:                Accepted
+      Status:                True
+      Type:                  Accepted
+      Last Transition Time:  2024-07-14T23:36:37Z
+      Message:               service configured with IPv4 family but NginxProxy is configured with IPv6
```

Comment:
Just noticing this, but our other Condition messages start with a capital letter. Let's update the message in the code and here to be `Service...`

Reply:
the linter complains on having `error strings capitalized`

Reply:
ah, ok.

Reply:
Looks like this string needs to capitalized now that you've changed the capitalization in the code.